python/GafferSceneUI/__init__.py

```diff
@@ -48,6 +48,7 @@
 from .FilterPlugValueWidget import FilterPlugValueWidget
 from .ScenePathPlugValueWidget import ScenePathPlugValueWidget
 from .LightEditor import LightEditor
+from .HistoryPathDialogue import HistoryPathDialogue
```

Comment:
I think this is in the wrong commit, but I'd also like to keep the HistoryPathDialogue out of the public API for now. Could we move it into LightEditor.py, or put it in `_HistoryPathDialogue.py` and import it directly from LightEditor?